app/src/main/java/org/wildaid/ofish/ui/basicinformation/BasicInformationFragment.kt

```diff
@@ -134,7 +136,24 @@ class BasicInformationFragment : BaseReportFragment(R.layout.fragment_basic_info
     }
 
     private fun onDatePicked(datePicker: DatePicker, year: Int, month: Int, dayOfMonth: Int) {
-        fragmentViewModel.updateDate(year, month, dayOfMonth)
+        val calendar: Calendar = Calendar.getInstance(TimeZone.getDefault())
```

Comment:
Date picker has mechanism to disable max date.. Please use it instead of notifications